model_checking/sail_prelude/lib.rs

```diff
@@ -667,7 +684,9 @@ mod tests {
 
         let mut v = BitVector::<SIZE>::new(0);
         let mut val: u64 = 0;
-        for idx in 0..SIZE {
+        for _ in 0..100 {
+            let idx = random::<usize>() % SIZE;
+
```

Comment:
`rand` was added back due to the divergence of the two `sail_prelude`. I can remove it in both versions if you want, just let me know.

Reply:
Let's keep it like that. One of the tests works with randomness.

Reply:
Well I'd prefer not to have randomness, that's why I am asking :)

Reply:
But randomness makes the test more robust. Otherwise I have a single pattern. Here I have 100 randoms patterns. Are you sure you don't want to keep it?

Reply:
Removed

Reply:
Thanks haha, I'm reviewing the PR and I'll merge it, looks great so far :)